MovieApp/LogInViewController.swift

```diff
@@ -0,0 +1,143 @@
+//
+//  LogInViewController.swift
+//  MovieApp
+//
+//  Created by endava-bootcamp on 28.03.2023..
+//
+
+import Foundation
+import UIKit
+import PureLayout
+import MovieAppData
+
+class LogInViewController: UIViewController {
+    
+    private var mainLabel: UILabel!
+    private var emailLabel: UILabel!
+    private var passLabel: UILabel!
+    private var emailField: UITextField!
+    private var passField: UITextField!
+    private var button: UIButton!
+    
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        buildView()
+    }
+    
+    func signInLabel(){
```

Comment:
Method name should be clear and state the intent. For example, `buildMainLabel` if you are keen on using methods for building UI elements. You have called this one `signInLabel`, but are building the `mainLabel`.

One way of doing this, that could be helpful, is to use `createViews` method to create the UI elements, labels and textFields for example, and add them in the view hierarchy. Then, use `styleViews` method to, well, style them. Add text color, text, fonts, etc. And lastly, use `defineLayoutForViews` method to add constraints.